src/options/arith_options.toml

```diff
@@ -398,7 +398,7 @@ name   = "Arithmetic Theory"
   category   = "regular"
   long       = "nl-ext-tplanes"
   type       = "bool"
-  default    = "false"
+  default    = "true"
```

Comment:
I find this not very transparent, given that whether these lemmas are actually used still depends on the value of `--nl-ext`. I'd rather extend the `--nl-ext` option to have an additional value that means `full + tplanes`.
Maybe we can also get rid of the interleaving option?

Reply:
In general I agree but I think this is a case where stratification is best, to prevent an exponential number of modes. Note that the same issue occurs with `--nl-ext-tf-tplanes`.

Interleaving actually meant run tangent planes at higher priority, e.g. in parallel with the other nl-ext-full schemes.  This was actually really helpful at one point, I'm not sure if the strategy refactoring preserved this, if not, we should fix this.

Reply:
I'm not sure I fully agree, but let's keep it for the moment.
The interleaving option is still used, and I think in the way you describe it (you might want to check the details, though).